<?xml version="1.0"?>
<launch>
    <arg name="map_file" default="$(find waypoint_nav)/../maps/cit_3f_map"/>
    <arg name="filename" default="$(find waypoint_nav)/../waypoints/route_a-f.yaml"/>
    <node pkg="map_server" type="map_server" name="map_server" args="$(arg map_file).yaml"/>

    <node pkg="waypoint_nav" type="waypoint_editor" name="waypoint_editor" output="screen">
        <param name="filename" value="$(arg filename)"/>
        <param name="map_frame" value="map"/>
        <rosparam file="$(find waypoint_nav)/config/function_list_detailed.yaml" command="load"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find waypoint_nav)/rviz/editor.rviz" />
</launch>